packages/@okta/i18n/src/properties/login.properties

```diff
@@ -318,6 +320,7 @@ mfa.redial = Redial
 mfa.sendEmail = Send email
 mfa.resendEmail = Re-send email
 mfa.scanBarcode = Scan barcode
+mfa.scanQrcode = Scan QR code
```

Comment:
instead of adding a new key, we should change the existing one `mfa.scanBarcode` since it is used in both places you need

Reply:
done